src/ferui-components/tree-view/tree-node-component.ts

```diff
@@ -30,7 +30,7 @@ import { ScrollbarHelper } from '../utils/scrollbar-helper/scrollbar-helper.serv
   selector: 'fui-tree-node',
   template: `
     <div class="fui-node-tree" (click)="onSelected()" [ngClass]="{ 'node-tree-selected': node.selected }">
-      <div [style.padding-left.px]="padding" class="node-tree" #nodetree>
+      <div [style.padding-left.px]="calculatePadding()" class="node-tree" #nodetree>
```

Comment:
Can't we use another way to bind this ? 
Because in Angular if you bind a function like this it will call it every-time instead of only during a changeDetection like with a variable (it is not like AngularJS where it doesn't matter because the digest loop runs every-time) and you are doing some calculation within your function... That's not ideal. 
I would bound this attribute to a simple variable and add an observable that will update the variable each time it needs too. It would be much more efficient that way.

Reply:
@JoMas971 you're right. I'm now just using the actual variable `padding` and instead of the observable i'm using the ngOnChanges lifecycle method for recalculating the padding if need be